automation.py

```diff
@@ -55,6 +65,35 @@ def automate_passovers():
             print("Scheduled to automate at the next passover.")
     else:
         print("AUTOMATION: no more passovers found.")
+        # calculate new passovers for the next 24 hours using current TLE data
+
+        hk = housekeeping.get(local_args={'limit': 1, 'newest-first': 'true'})
+
+        tle = hk[0]['data']['logs'][0]['tle']
+        lines = tle.split('\n')
+    
+        orb = Orbital('ex-alta 2', line1=lines[0], line2=lines[1])
+        dtobj = datetime.utcnow()
+        passes = orb.get_next_passes(dtobj, 24, -113.4938, 53.5461, 0.645) # edmonton coordinates and elevation
+        ps_data = {'passovers': [{'timestamp': str(ps[0])} for ps in passes]}
+
+        passover.post(local_data=json.dumps(ps_data))
+        
+
+def send_slack_notifs(message):
+    # api call to get all users
+    user_list = UserList()
+
+    users = user_list.get(local_args={'limit': 1000})[0]['data']
+
+    client = slack.WebClient(token='xoxb-58810056594-3097149250131-xK3JH7caXxC5oQ9aQ0etzAsM')
```

Comment:
The exposed API key issue this repos has may need to be addressed soon. Slack [might be proactively looking for exposed keys on GitHub](https://labs.detectify.com/2016/04/28/slack-bot-token-leakage-exposing-business-critical-information/) and deactivating them.

Reply:
Removed the slack token for now and just called for an environment variable.